pages/about.html

```diff
@@ -0,0 +1,154 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Architectural Summit | About</title>
+    <link rel="stylesheet" href="../css/style.css" />
+  </head>
+  <body>
+    <header>
+      <ul class="social">
+        <li>
+          <a href="http://web.facebook.com/lawrence.amoafo"
+            ><ion-icon name="logo-facebook" id="icon"></ion-icon
+          ></a>
+        </li>
+        <li>
+          <a href="http://twitter.com/lawrence_amoafo"
+            ><ion-icon name="logo-twitter" id="icon"></ion-icon
+          ></a>
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+
+      <nav class="head">
+        <div class="vector">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+
+        <div class="logo">
+          <a href="../index.html"
+            ><img src="../Images/logo.png" alt="logo"
+          /></a>
+          <a href="../index.html"><b>World Architectural Summit 2023</b></a>
+        </div>
+
+        <ul class="nav-links">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+        </ul>
+
+        <ul class="nav-links2">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+          <li class="style">WA Summit</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="about">
+        <h1 class="about-heading">WORLD ARCHITECTURAL SUMMIT 2023</h1>
+        <p class="about-paragraph">
+          World Architectural Summit will build on the breakout success of the
+          inaugural edition with a captivating and insightful agenda that will
+          be discussed and debated by some of the region's best and brightest
+          minds. The Summit will provide a platform for construction industry
+          stakeholders to discuss a multitude of topics including future city
+          requirements, net zero building design, resilient design strategies,
+          future transport infrastructure, nurturing future talent and much
+          more. Along with a rich agenda, the summit will also provide
+          unparalleled networking opportunities in full compliance with all
+          local COVID-19 related regulations.
+        </p>
+      </section>
+
+      <section class="location" id="location">
+        <h2 class="location-heading">Location</h2>
+        <div class="line"></div>
+        <p class="location-paragraph">
+          World Architectural Summit will take place in <br /><span
+            ><b>THE OBEROI / DUBAI / UAE on 1st JUNE 2023</b></span
+          ><br />
+          in full compliance with all COVID-19 safety guidelines and protocols.
+        </p>
```

Comment:
- [ ] Kindly consider following the Original Behance design for font-size, font-weight and the font colors in this section. your font-sizes seem bigger than what in the Original design. also, the paragraph text has lighter color in the Original design so please try fixing it. 
- [ ] Here in this section, instead of this line `<b>THE OBEROI / DUBAI / UAE on 1st JUNE 2023</b>`. consider adding the logo or image with thin border as in the original design.
your design
![vc](https://user-images.githubusercontent.com/96683843/226000895-d857f6d5-4f2a-44b0-94bb-2971477e71e6.jpg)


Original Design
![zx](https://user-images.githubusercontent.com/96683843/226000921-cde2cdd0-3f51-414d-912b-22e39d723172.jpg)